manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cookiebot",
-  "vendor": "vtex",
-  "version": "2.0.7",
+  "vendor": "arcaplanetqa",
+  "version": "0.0.0",
```

Comment:
version 0.0.0 cannot be published, this will need to be changed

Reply:
Hi @Git-the-Sanz , Thanks for looking into it!

I completely agree with you, that's why I have already changed the version on [this commit](https://github.com/vtex-apps/cookiebot/pull/13/commits/1381dd1de4f41c550bb65e9710666dc95f90fb07)

I apologize for the confusion caused by editing the manifest in 2 different commits, I started to commit while developing the app and at that time I didn't thought that I might need to open a PR with them! It was done also in a bit of an hurry.

Also, if I need to commit the version change again, or do anything that could ease the merge, just let me know!

If instead, I am completely wrong, and I've mixed up the commits order, sorry also for not getting it, and again, just le me know what I should change and I will do it right away!

Thanks!